borsh-rs/borsh/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "borsh"
-version = "0.7.0"
+version = "0.8.0"
```

Comment:
Why do you update the minor version? According to semver, releases before 1.0 treated differently and minor version change indicates a breaking change. In this case, it is a new feature, so we can just bump the patch version. /cc @chefsale

Reply:
I see.

Reply:
Fixed